rpctest/02_validateaddress_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2018 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+package rpctest
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/decred/dcrd/chaincfg"
+	"github.com/decred/dcrd/dcrutil"
+	"github.com/decred/dcrd/txscript"
+	"github.com/decred/dcrd/rpcclient"
+)
+
+func TestValidateAddress(t *testing.T) {
+	// Skip tests when running with -short
+	if testing.Short() {
+		t.Skip("Skipping RPC harness tests in short mode")
+	}
+	if skipTest(t) {
+		t.Skip("Skipping test")
+	}
+	r := ObtainHarness(MainHarnessName)
+	// Wallet RPC client
+	wcl := r.WalletRPCClient()
+
+	accounts := []string{"default", "testValidateAddress"}
+
+	for _, acct := range accounts {
+		// Create a non-default account
+		if strings.Compare("default", acct) != 0 &&
+			strings.Compare("imported", acct) != 0 {
+			err := r.WalletRPCClient().CreateNewAccount(acct)
+			if err != nil {
+				t.Fatalf("Unable to create account %s: %v", acct, err)
+			}
+		}
+
+		// Get a new address from current account
+		addr, err := wcl.GetNewAddressGapPolicy(
+			acct, rpcclient.GapPolicyIgnore)
```

Comment:
Seems like address requested with `GapPolicyIgnore` fails the `IsMine` check at line 61. Is it a bug or expected behavior?